src/Integration/LocalServices/IErrorListInfoBarController.cs

```diff
@@ -0,0 +1,22 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IErrorListInfoBarController.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace SonarLint.VisualStudio.Integration
+{
+    internal interface IErrorListInfoBarController : ILocalService
+    {
+        /// <summary>
+        /// Checks whether the error list info bar needs to be displayed
+        /// </summary>
+        void Refresh();
+
+        /// <summary>
+        /// Clears the info bar, if any
+        /// </summary>
+        void Reset();
```

Comment:
Wasn't immediately obvious to me what `Refresh` and `Reset` do.
So `Reset` 'clears' the info bar, does that mean _hide_ it?
What does `Reset` do then when it is decided that it no longer needs to be displayed? Does it hide it? Update a property/event (which aren't on the interface)?